### DISCUSSION TITLE
Model skeletal animation

### DISCUSSION THREAD

**Discussion by Snoopy137** [AlmasB/FXGL#1256] [Q&A]
So I am experimenting with FXGL and I created a 3d model on blender and have animated but I could not find the way to apply it into teh game, I can import the static 3d model but cannot use the animation.
Does any one have done this or knows an example for this?

**Comment by chengenzhao** [comment]
same here
I think we should create [Animated Transformation](https://github.com/AlmasB/FXGL/discussions/1246) first 
then make a Entity Builder editor to allow developers visually create and composite different 3D or 2D components
and finally composite these components

**Reply by Snoopy137** [reply]
Hey, I tried to open the launcher but looks like with the zip file some dependencies are missing
